src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -239,19 +239,19 @@ public void CanCreateEnvironmentWithAllProperties()
             {
                 CommandRuntime = commandRuntimeMock.Object,
                 Name = "Katal",
-                ActiveDirectoryEndpoint = "ActiveDirectoryEndpoint",
+                ActiveDirectoryEndpoint = "https://ActiveDirectoryEndpoint",
```

Comment:
@cormacpayne it will be a good idea to add one more test case that actually adds trailing slash at the end, especially we are fixing a trailing slash issue, plus this would be a good test to find out if we have any errors when there multiple // slash in an url.